assets/docs/steinding/models/yolov5/1.md

```diff
@@ -0,0 +1,51 @@
+# Module steinding/yolov5/1
+
+Object detection model pretrained on YoloV5. 
+<!-- asset-path: https://drive.google.com/drive/folders/1UGiX2CpMVpgRZY1FsdfjOlV9LU9An2HG?usp=sharing -->
+<!-- task: Object-detection -->
+<!-- network-architecture: regnety -->
+<!-- format: saved_model_2 -->
+<!-- fine-tunable: true -->
+<!-- license: Anti 996-->
```

Comment:
Could you please choose a key from https://github.com/tensorflow/tfhub.dev/blob/master/tags/license.yaml? If none matches, please choose "custom" and copy the license to the bottom of this file.

Reply:
Done, changed to custom

Reply:
Please also copy the license that applies to the bottom of the page

Reply:
License content is now added to the bottom